pom.xml

```diff
@@ -128,6 +129,16 @@
 
 	<dependencyManagement>
 		<dependencies>
+			<dependency>
```

Comment:
Why do we have a direct dependency on bytebuddy now?

Reply:
Non-converging otherwise.